diff --git a/node_modules/notion-utils/build/index.js b/node_modules/notion-utils/build/index.js
index 5e45758..299c23f 100644
--- a/node_modules/notion-utils/build/index.js
+++ b/node_modules/notion-utils/build/index.js
@@ -530,7 +530,9 @@ var normalizeTitle = (title) => {
 };
 
 // src/uuid-to-id.ts
-var uuidToId = (uuid) => uuid.replaceAll("-", "");
+// var uuidToId = (uuid) => uuid.replaceAll("-", "");
+var uuidToId = (uuid) => uuid?.replaceAll("-", "") ?? "";
+
 
 // src/get-canonical-page-id.ts
 var getCanonicalPageId = (pageId, recordMap, { uuid = true } = {}) => {
